Gds/src/fprime_gds/common/utils/config_manager.py

```diff
@@ -11,7 +11,7 @@
 
 @date Created July 25, 2018
 
-@lisence Copyright 2018, California Institute of Technology.
+@license Copyright 2018, California Institute of Technology.
```

Comment:
Changing license blocks occasionally results in devs getting scared.

`license` and `licence` are both words, `lisence` isn't.

That said, in the US, the former is used for both things -- https://www.grammarly.com/blog/licence-license/

Reply:
This is fine.